src/site/_includes/components/Export.js

```diff
@@ -64,6 +64,18 @@ async function Export() {
     return '';
   }
 
+  const authors = JSON.stringify(page?.template?.frontMatter?.data.authors);
+
+  const frontMatter = `project_path: /_project.yaml
+book_path: /_book.yaml
+
```

Comment:
There is also `author_name`, used in dynamic content cards.

Reply:
true, we can add it